content/en/blog/_posts/the-case-for-kubernetes-limits/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: "The Case for Kubernetes Resource Limits: Predictability vs. Efficiency"
+date: TODO
+slug: the-case-for-kubernetes-resource-limits-predictability-vs-efficiency
+---
+
+**Author:** Milan Plžík
+
+
+
+_Milan Plzik is a Site Reliability Engineer on the [Grafana Labs](http://grafana.com) platform team, which maintains and improves internal infrastructure and tooling used by the product teams. As part of the infrastructure squad, he is primarily trying to make Kubernetes upgrades as smooth as possible. He likes going down the rabbit hole of various interesting Kubernetes issues._
+
+_Disclaimer: This article reflects the author's personal opinion; feel free to disagree. Also, there are a lot of subjective ideas that have not made it through any kind of rigorous testing. Use at your own risk._
```

Comment:
We don't usually include either kind of a paragraph. If it's essential that we do, I'd prefer to boilerplate it so that all articles where its needed get the exact same text.

However, please do state or hint in the running text that these are your opinions and that different people in the end user community may have their won.

Reply:
Updated; PTAL.